### ISSUE TITLE
When margins are set - MS Word Error

### ISSUE THREAD

**Issue by 4got** [privateOmega/html-to-docx#57]
Hi, default margins are bad for me, so i use this
```
margins: {
  top: '1cm',
  right: '1.5cm',
  bottom: '2cm',
  left: '2.5cm',
},
```

but
MS Word shows an Error

![image](https://user-images.githubusercontent.com/32334252/108897443-287cc600-7627-11eb-8f9c-a4e0268e09aa.png)
it is said: "Open the file with reserve copies converter / file is corrupted"

only Google Docs could open
help pls!

**Comment by privateOmega** [comment]
yes sorry, I will fix the margin merging, so that you won't have to explicitly provide other margins if you need to change only certain values. @iagomontenegrob 's proposed answer should work for you.

**Comment by mkoskl** [comment]
@privateOmega what's the status of this?
